src/pages/integrations/coinbase/coinbase.ts

```diff
@@ -130,28 +130,30 @@ export class CoinbasePage {
   }
 
   public openAuthenticateWindow(): void {
-    let oauthUrl = this.getAuthenticateUrl();
-    if (!this.isNW) {
+    const oauthUrl = this.getAuthenticateUrl();
+    if (!this.isElectron) {
       this.externalLinkProvider.open(oauthUrl);
     } else {
-      let gui = (window as any).require('nw.gui');
-      gui.Window.open(
-        oauthUrl,
-        {
-          focus: true,
-          position: 'center'
-        },
-        new_win => {
-          new_win.on('loaded', () => {
-            let title = new_win.window.document.title;
-            if (title.indexOf('Coinbase') == -1) {
-              this.code = title;
-              this.submitOauthCode(this.code);
-              new_win.close();
-            }
-          });
+      const { remote } = (window as any).require('electron');
+      const BrowserWindow = remote.BrowserWindow;
+      const win = new BrowserWindow({
+        alwaysOnTop: true,
+        center: true,
+        webPreferences: { nodeIntegration: false }
+      });
+      win.once('ready-to-show', () => {
+        win.show();
+        win.focus();
+      });
+      win.loadURL(oauthUrl);
+      win.webContents.on('did-finish-load', () => {
+        const title = win.webContents.getTitle();
+        if (title.indexOf('Coinbase') == -1) {
```

Comment:
lets use something different than the tile to identify the window here. Maybe add some metadata or an extra attribute.

Reply:
Currently, that is the easiest way to identify that the title has the auth code. But, we could plan a refactor for later.